api/inventorypb/agents.proto

```diff
@@ -629,6 +650,8 @@ message AddPostgresExporterRequest {
   map<string, string> custom_labels = 7;
   // Skip connection check.
   bool skip_connection_check = 8;
+  // Enables push metrics mode for exporter.
+  bool push_metrics = 9;
 }
```

Comment:
It should also be added to proxysql_exporter, postgres_exporter rds_exporter at least. But please also check with @BupycHuk other types.

Reply:
as far as i understand, there is no need for rds_exporter and it will be scrapped by pmm-server. 

 for proxysql_exporter needs guidance, dont know what it is.

Reply:
All those exporters are in this file.
rds_exporter is _typically_ started on PMM Server, but it is a normal exporter and it _can_ be started anywhere, so no reason to exclude it.

Reply:
added missing exporters.